cmd/create/cluster/cmd.go

```diff
@@ -3914,26 +3914,39 @@ func getExpectedResourceIDForAccRole(hostedCPPolicies bool, roleARN string, role
 	return strings.ToLower(fmt.Sprintf("%s-%s-Role", rolePrefix, accountRoles[roleType].Name)), rolePrefix, nil
 }
 
-func getInitialValidSubnets(aws aws.Client, ids []string, r *reporter.Object) ([]*ec2.Subnet, error) {
+func getInitialValidSubnets(awsClient aws.Client, ids []string, r *reporter.Object) ([]*ec2.Subnet, error) {
```

Comment:
@marcolan018 Could you please add a unit test for this function. You will need to generate a `mock` of the AWSClient, or wait for #1742 to be merged where @thomasmckay is also generating a mock for the client.